conformance/submission_details_template.txt

```diff
@@ -0,0 +1,105 @@
+##############################################################################
+#
+# SYCL Conformance Submission Details Template
+# Source: https://github.com/KhronosGroup/SYCL-CTS
+# Version: 1.0
+#
+##############################################################################
+#
+# This form should be filled in and included in the conformance submission
+# as the file submission_details.txt.
+#
+# Lines beginning with "#" are comments to assist in filling out the template.
+#
+##############################################################################
+
+##############################################################################
+#
+# Submission information that needs to be copied onto the web submission form.
+#
+##############################################################################
+
+# Contact details.
+#
+# Contact details for individual making the conformance submission.
+#
+Company: 
+Contact name:
+Contact email:
+Contact telephone:
+
+# Version of the SYCL specification being tested.
+#
+# Specification version of SYCL being tested, can be either 1.2.1 or SYCL.
```

Comment:
Should this say `or SYCL 2020`?

Reply:
Or just "or 2020"

Reply:
Yes that should be `2020` thanks, fixed in https://github.com/KhronosGroup/SYCL-CTS/pull/803/commits/a230561300bb379a65616c5ab38ec8bea2886fda.